### PR TITLE
Addition of Vgp0 : assembly of mitochondrial genome

### PR DESCRIPTION
**Pull request by Delphine-L**

### CODE REVIEW

File: workflows/VGP-assembly-v2/Mitogenome-assembly-VGP0/Mitogenome-Assembly-VGP0-tests.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  job:
+    Collection of Pacbio Data:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: pacbio_01.fasta.gz
+        location: https://zenodo.org/records/10454765/files/pacbio_01.fasta.gz?download=1
+    Species name (latin name): Theretra latreillii lucasii
+    Email adress: lariviere.delphine@gmail.com
```
[the remark this answers is not in the document]
Comment (Delphine-L) [reply workflows/VGP-assembly-v2/Mitogenome-assembly-VGP0/Mitogenome-Assembly-VGP0-tests.yml:11]: No way around i think. Should we use a galaxy email address instead ?

[the remark this answers is not in the document]
Comment (mvdbeek) [reply workflows/VGP-assembly-v2/Mitogenome-assembly-VGP0/Mitogenome-Assembly-VGP0-tests.yml:11]: I guess that would make sense ? iwc@galaxyproject.org ?

File: workflows/VGP-assembly-v2/Mitogenome-assembly-VGP0/Mitogenome-Assembly-VGP0.ga [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+        {
+            "class": "Organization",
+            "name": "Galaxy"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "CC-BY-4.0",
+    "version":"0.1",
+    "name": "Mitogenome-Assembly-VGP0",
+    "steps": {
```
Comment (mvdbeek) [inline workflows/VGP-assembly-v2/Mitogenome-assembly-VGP0/Mitogenome-Assembly-VGP0.ga:20]:
```suggestion
    "release": "0.1",
    "steps": {
```

File: workflows/VGP-assembly-v2/Mitogenome-assembly-VGP0/Mitogenome-Assembly-VGP0.ga [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+            "url": "https://vertebrategenomeproject.org"
+        },
+        {
+            "class": "Organization",
+            "name": "Galaxy"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "CC-BY-4.0",
+    "version":"0.1",
```
Comment (mvdbeek) [inline workflows/VGP-assembly-v2/Mitogenome-assembly-VGP0/Mitogenome-Assembly-VGP0.ga:18]:
```suggestion
```

**Review by mvdbeek — APPROVED** [review]

**Pull request merged**